taca/storage/storage.py

```diff
@@ -44,23 +51,30 @@ def archive_to_swestore(config, days, run=None):
     # Otherwise find all runs in every data dir on the nosync partition
     else:
         LOG.info("Archiving old runs to SWESTORE")
-        for data_dir in config.get('storage').get('data_dirs'):
-            to_send_dir = os.path.join(data_dir, 'nosync')
+        for to_send_dir in config.get('storage').get('archive_dirs'):
             LOG.info('Checking {} directory'.format(to_send_dir))
             with filesystem.chdir(to_send_dir):
-                for run in [r for r in os.listdir(to_send_dir) if re.match(filesystem.RUN_RE, r)]:
-                    _archive_run(config, run)
-
+                to_be_archived = [r for r in os.listdir(to_send_dir) if re.match(filesystem.RUN_RE, r)
+                                            and not os.path.exists("{}.archiving".format(r.split('.')[0]))]
+                if to_be_archived:
+                    pool = Pool(processes=len(to_be_archived))
+                    pool.map_async(_archive_run, ((run,) for run in to_be_archived))
+                    pool.close()
+                    pool.join()
```

Comment:
Awesome :+1:  Have always wondered about this one i.e. to split some things to multiple core within the same script. Now I know, Tack @guillermo-carrasco :). 

This should work for us as we have 24 cores and it is close to 0% chance that we archive 24 runs at a time, so we are fine but I have a general curiosity. For example, if we have only 4 cores and the `len(to_be_archive)` is 6, than what will happen?  would it complain or it will just run 4 entires from `to_be_archived` ?